Demo/__init__.py

```diff
@@ -0,0 +1,7 @@
+class QueryInfo(object):
+
+    def __init__(self, query=None, description=None, max_run_time=None, expected_result=None):
```

Comment:
Please add comments describing the data types foreach property

Reply:
done